avocado/plugins/xunit.py

```diff
@@ -47,11 +47,15 @@ def _escape_cdata(self, cdata):
     def _get_attr(self, container, attrib):
         return self._escape_attr(container.get(attrib, self.UNKNOWN))
 
+    @staticmethod
+    def _format_time(time):
+        return "{:.3f}".format(float(time))
```

Comment:
well the old format is more common in Avocado, would you mind changing this to `"%.3f" % float(time)`? IMO it looks better plus it's about 2% faster on py3 and 30% faster on py2.